server/Utilities/EmailValidation.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.IdentityModel.Tokens;
+using server.DTO;
+using server.Models.DB;
+using server.Services;
+using System.ComponentModel.DataAnnotations;
+using System.Text.RegularExpressions;
+using Microsoft.AspNetCore.Authorization;
+
+
+namespace server.Utilities
+{
+
+    public static class EmailValidation
+    {
+        public static bool validate(string email)
```

Comment:
I believe this validation can be done on the client side itself , and this utility is not being used anywhere in current PR . 
Please feel free to remove this utility .

Reply:
Frontend would be fine too,, removed as requested